File: autoskola-central-front/src/app/core/models/polaznik.ts
import { Instruktor } from "./instruktor";
import { Korisnik } from "./korisnik";
import { Sat } from "./sat";
import { Vozilo } from "./vozilo";

export class Polaznik{

    polaznikId !: number ;
    instruktorId : number | undefined;
    korisnikId : number | undefined;
    statusPolaznikaId: number |undefined;
    odabranaKategorijaId: number |undefined;

    korisnik : Korisnik  = new Korisnik();
    instruktor : Instruktor | undefined;
    dnevnikVoznje !: Sat[];
}

File: autoskola-central-back/src/main/java/hr/autoskola/model/Polaznik.java
package hr.autoskola.model;

import java.util.List;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

@AllArgsConstructor
@NoArgsConstructor
@Getter
@Setter
public class Polaznik {
	
	private Integer polaznikId;
	private Integer korisnikId;
	private Integer instruktorId;
	private Integer statusPolaznikaId;
	private Integer odabranaKategorijaId;
	
	private Korisnik korisnik;
	private Instruktor instruktor;
	private List<Sat> dnevnikVoznje; 

}
